packages/steam-deb/steam-deb.pacscript

```diff
@@ -0,0 +1,8 @@
+name="steam-deb"
+gives="steam"
```

Comment:
```suggestion
gives="steam"
breaks="${gives}-git ${gives}-bin ${gives}-app"
```

Reply:
I don't think Steam provides any of these

Reply:
If a package comes along in the future that is a binary or appimage, this is gonna be needed.